README.md

```diff
@@ -120,6 +120,30 @@ and tactics specialized for monadic programs (see
 
 A tutorial for the Lean backend is available [here](./tests/lean/Tutorial.lean).
 
+## Quick start for Nix users
+
+Assuming Nix is installed, with a support for Flakes (`*`):
+
+```console
+$ nix run github:aeneasverif/aeneas#charon -L
+$ # The default package for Aeneas is Aeneas.
```

Comment:
I think this comment is more confusing than not having it :p

Reply:
Removed.